Jina credential validation POST lacks bounded timeout

### Self Checks

- [x] I have searched for existing issues and pull requests and did not find an open report for Jina auth credential-validation timeouts.
- [x] I verified the behavior locally with a focused failing test before filing.

### Dify version

main branch

### Steps to reproduce

`services.auth.jina.jina.JinaAuth.validate_credentials()` validates a Jina Reader API key through `_post_request()`, which currently delegates to the pooled HTTP client without an explicit timeout:

```python
def _post_request(self, url, data, headers):
    return _http_client.post(url, headers=headers, json=data)
```

I verified this with a focused unit-test assertion that the request should include a bounded timeout:

```bash
uv run --project api pytest -o addopts='' api/tests/unit_tests/services/auth/test_jina_auth.py::TestJinaAuth::test_should_validate_valid_credentials_successfully -q
```

The test fails on current `main` because the `timeout` keyword is missing:

```text
E       KeyError: 'timeout'
```

### Expected Behavior

Jina credential validation should pass an explicit bounded `httpx.Timeout` to the outbound POST request. A slow or hanging Jina endpoint should not be able to stall credential validation indefinitely or for an implementation-defined default.

### Actual Behavior

The Jina auth provider calls `_http_client.post(url, headers=headers, json=data)` without a timeout. Nearby outbound provider and remote retrieval calls now use explicit timeout constants such as `httpx.Timeout(10.0, connect=3.0)`.

### Additional context

This is the Jina auth-provider equivalent of the bounded timeout hardening recently applied to adjacent auth/provider paths. A minimal fix is to define a small `httpx.Timeout` constant in `services/auth/jina/jina.py` and pass it to `_post_request()`.

--- a/api/services/auth/jina.py
+++ b/api/services/auth/jina.py
@@ -8,7 +8,10 @@ from services.auth.api_key_auth_base import ApiKeyAuthBase, AuthCredentials
 
 _http_client: httpx.Client = get_pooled_http_client(
     "auth:jina_standalone",
-    lambda: httpx.Client(limits=httpx.Limits(max_keepalive_connections=50, max_connections=100)),
+    lambda: httpx.Client(
+        timeout=httpx.Timeout(10.0),
+        limits=httpx.Limits(max_keepalive_connections=50, max_connections=100),
+    ),
 )
 
 

--- a/api/services/auth/jina/jina.py
+++ b/api/services/auth/jina/jina.py
@@ -8,7 +8,10 @@ from services.auth.api_key_auth_base import ApiKeyAuthBase, AuthCredentials
 
 _http_client: httpx.Client = get_pooled_http_client(
     "auth:jina",
-    lambda: httpx.Client(limits=httpx.Limits(max_keepalive_connections=50, max_connections=100)),
+    lambda: httpx.Client(
+        timeout=httpx.Timeout(10.0),
+        limits=httpx.Limits(max_keepalive_connections=50, max_connections=100),
+    ),
 )
 
 

